backend/rebutify/settings.py

```diff
@@ -48,14 +48,16 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    "corsheaders",
     "core",
     "rest_framework",
     "rebutify",
-    "crispy_bootstrap4",
-    "crispy_forms",
```

Comment:
Due to the decision to make an exception to trunk-based dev for feat/1_auth, we're gonna have to remove this and any other fixes already implemented in another branch.

These changes are already implemented in feat/1_auth.

Also, this breaks build since the templates rely on this dependency